packages/malloc/package.json

```diff
@@ -19,6 +19,8 @@
         "build:es6": "tsc --declaration",
         "build:test": "rimraf build && tsc -p test/tsconfig.json",
         "test": "yarn build:test && mocha build/test/*.js",
+        "t": "TS_NODE_PROJECT=test/tsconfig.json node -r ts-node/register ./node_modules/.bin/mocha test/*.ts",
```

Comment:
Just out of interest: what does this achieve? Running tests via ts-node?

Reply:
run tests without need to build the test files first, with sourcemaps (see real errors locations in the ts, and jump to them), and and "td" command also let you debug the tests
works like a charm with vscode autoattach

BTW, with jest the DX will be even better:)

Reply:
I will not include these changes on that PR, but they helped me a lot

Reply:
Thanks, that's very interesting and will need to read up. I know this project (well, I am) not too up-to-date in terms of JS/TS magic tooling, but happy to adapt unless its causing huge workload/changes. But in general, that should probably be done across the board then and in a separate PR... :)